docs/tutorial/using-checkbox/basic-launcher.rst

```diff
@@ -0,0 +1,260 @@
+- in the ``[test selection]`` section, we forced the selection of all the
+  tests, bypassing the test selection screen.
+
+Customize test cases with environment variables
+===============================================
+
+One of the test cases in the Tutorial test plan uses the value set in an
+environment variable. Add the following lines in the launcher:
+
+.. code-block:: none
+    :caption: mylauncher
+    :name: launcher-environment
+    :emphasize-lines: 12-13
+
+    [launcher]
+    launcher_version = 1
+    app_id = com.canonical.certification:tutorial
+
+    [test plan]
+    unit = com.canonical.certification::TODO
+    forced = yes
+
+    [test selection]
+    forced = yes
+
+    [environment]
+    TUTO = tutorial
+
+Run Checkbox using your launcher, and observe the output of the TODO test
+case. The output now shows ``tutorial``.
+
+The ``[environment]`` section is often used to provide customized values to
+test cases. For instance, you may have a generic test case to connect to a
+WiFi access point, but its SSID and password might change, so you can use an
+environment variable in the test case definition and set their values in the
+``[environment]`` section of your launcher.
+
+Tailor Checkbox output
+======================
+
+At the end of the test session, Checkbox summarizes the test results on
+the screen, generates test reports and test archive, and asks you whether
+you want to upload the test results to the Canonical :term:`Certification
+website`. Let's say you don't need to upload the results there; you are only
+interested in the text summary and the test reports.
+
+Edit the launcher file:
+
+.. code-block:: none
+    :caption: mylauncher
+    :name: launcher-stock-reports
+    :emphasize-lines: 4
+
+    [launcher]
+    launcher_version = 1
+    app_id = com.canonical.certification:tutorial
+    stock_reports = text, submission_files
+
+    [test plan]
+    unit = com.canonical.certification::TODO
+    forced = yes
+
+    [test selection]
+    forced = yes
+
+    [environment]
+    TUTO = tutorial
+
+Run Checkbox using this launcher and observe that once the test plan is
+finished running, Checkbox generates a summary on the screen and provides
+the links to the test reports and test archive, but does not ask if the
+result should be uploaded to the Canonical Certification website.
+
+This is thanks to the customization of the ``stock_reports`` field in the
+``[launcher]`` section. If not specified in the launcher, its default value
+is set to ``text, certification, submission_files``.
+
+In Checkbox language, submissions files are the HTML test report as well as
+an archive containing the test results and additional logs that might have
+been produced by the test cases.
+
+Create an executable launcher
+=============================
+
+So far, we have called our launcher using the ``launcher`` option of the
+``checkbox-cli`` tool. It is however possible to turn our launcher into a
+"standalone" executable.
```

Comment:
interpretable file similarly to a bash script

definitely not standalone

Reply:
Thanks, will modify